package.json

```diff
@@ -27,6 +27,7 @@
     "react-reveal": "^1.2.2",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.0",
+    "react-testing-library": "^8.0.1",
```

Comment:
`@testing-library/react` is used in place of `react-testing-library` now, so the `react-testing-library` package can be removed